package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-draggable",
-  "version": "3.0.5",
+  "version": "3.0.5-pl.0",
```

Comment:
Please do not change the version string.

Reply:
Sorry, I committed that by mistake. It was only for testing purposes.